docs/RESOURCES.md

```diff
@@ -1,13 +1,7 @@
 # Resources
 
-<!-- TODO: write document
-
-  This document should link to useful resources.
-
-  Feel free to link to any good resources you know, whether they
-  be websites, videos, mailing lists, etc.
-
-  The contents of this document are displayed on the track's documentation
-  page at `https://exercism.org/docs/tracks/<track>/resources`.
-
-  See https://exercism.org/docs/building/tracks/docs for more information. -->
+* [Godot Engine - Q&A](https://ask.godotengine.org/)
```

Comment:
@BNAndras should this list start with the main Godot Engine's website (https://godotengine.org/), or will it be included somewhere else?

Reply:
This file is for useful GDScript resources in general so why not? I probably didn't include it as an oversight since I was already on the website in question. Were you thinking a link to the home page? Perhaps a link to the docs section at https://docs.godotengine.org/en/stable/?

Reply:
I was thinking about the main website (https://godotengine.org/), as it leads you to multiple download options (normal vs mono, different platforms, v3 vs v4), and there is also a blog with many useful updates and a link to the asset library. But now that you've mentioned it, I think that adding a direct link to the documentation is a great idea! Do you think we could have both, or would it be too much?

BTW, maybe a link to the asset library could also be added here?

Reply:
Are the assets usable within GDScript or Godot at large? I think the focus should be on GDScript although since it's tightly integrated into Godot, any divide between the two might be murky.

Reply:
Hm ... that depends. I think most of the resources in Asset Library contain at least a bit of GDScript, but they often specialize in 2D scenes, 3D scenes, shaders etc. So, perhaps (at least for now) Godot's main website and current docs page would be enough?

Reply:
Sounds good. I'll update the PR over the weekend.